go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/tensorleap/leap-cli
 
-go 1.20
+go 1.21
 
```

Comment:
don't :)

Reply:
Hod do I avoid it?
If I don't run `go mod tidy` I can't navigate to imported packages and If I do run it bump version